configuration/white_labels/nc.py

```
@@ -493,53 +493,28 @@ def get_white_label(self) -> WhiteLabel:
+        },
+        "healthcare": {
+            "medical": {"_label": "expenseOptions.medical", "_default_message": "Medical Insurance Premium &/or Bills"},
         },
         "dependentCare": {
-            "_label": "expenseOptions.dependentCare",
-            "_default_message": "Dependent Care",
+            "childCare": {"_label": "expenseOptions.childCare", "_default_message": "Child Care"},
+            "childSupport": {"_label": "expenseOptions.childSupport", "_default_message": "Child Support (Paid)"},
+            "dependentCare": {"_label": "expenseOptions.dependentCare", "_default_message": "Dependent Care"},
         },
     }
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)